css/utilities/width.css

```diff
@@ -1,7 +1,19 @@
+.max-width-100pct {
```

Comment:
it could be `.max-width-full` and `width-full`

Reply:
I have made a PR in my CL and used `max-w-100pct` and `w-100pct` there, as I have other width classes in number too and not in `xs` or `s`. I have other classes like `w-32` for `width: 32rem`. So to keep consistency I am using numbers everywhere.